### ISSUE TITLE
Systemwide gestue to refresh doesn't work

### ISSUE THREAD

**Issue by heull001** [haiwen/seafile-iOS#316]
The systemwide gestue to refresh scrollable content doesn't work in the seafile-app. This is especially bad for bind people using voiceover on iOS, because the refresh-method in the app is not easy to use.

I can't explain how to reproduce without using voiceover, because I'm blind and don't know the normal gestue. While using voiceover (settings -> accessibility -> voiceover) I swipe downwards with three fingers and expect the app to refresh, but nothing happens.

Please see https://developer.apple.com/documentation/uikit/uirefreshcontrol for information to include the standard-gestue.

**Comment by Neurrone** [comment]
Seconded, hoping this can be fixed.

**Comment by heull001** [comment]
Is there any plan to change this? This is really annoying.

**Comment by freeplant** [comment]
We will look into the problem.

**Comment by heull001** [comment]
This was fixed by last update, thanks!
